roles/gitops/rendering-apps-files/templates/observability/Chart.yaml.j2

```
@@ -0,0 +1,6 @@
+---
+apiVersion: v2
+name: observability
+type: application
+version: 1.0.0
```

```suggestion
version: {{ dsc.observatorium.chartVersion }}
```
Pour que la version match celle du chart ? (et aussi avoir un truc en jinja dans le fichier).

---

À rediscuter dans la PR à venir sur Grafana